### ISSUE TITLE
The performance gap of PointCNN on TU-Berlin

### ISSUE THREAD

**Issue by KwanBL** [yangyanli/PointCNN#122]
Based on the commands(cat) about tu_berlin in README, the result is about 60%, while 67.72% in the arXiv version and 70.57% in the NIPS version.

**Comment by burui11087** [comment]
Hi @KwanBL 

Did you use tensorboard to check the performance and find that after many iterations, there is no increase?

**Comment by KwanBL** [comment]
@burui11087 
Yes, I did. It reached 60% after 50.00K iterations and it no longer increased even after 100.00K iterations.

**Comment by burui11087** [comment]
Hi @KwanBL 

I ran Tu_belin again and didn't change any thing based on this github version, it could reach 68.91% at 39.00K iteration。
![image](https://user-images.githubusercontent.com/10089180/50389319-e5cbb480-0763-11e9-9e62-e068f5612699.png)

So I want to know that did you change the hyperparameters and your GPU version.

Thanks

**Comment by jiaxinchenX** [comment]
Hi @burui11087  I have the same problem on TU-Berlin dataset. I ran the code based on your github version as follows :
 
"cd data_conversions
python3 ./download_datasets.py -d tu_berlin
python3 ./prepare_tu_berlin_data.py -f ../../data/tu_berlin/svg -a --create-train-test
cd ../pointcnn_cls
./train_val_tu_berlin.sh -g 0 -x tu_berlin_x3_l4"

However, the validation accuracy reached 60% at 40000 iterations, and was no longer decreased. The training/validation loss and training/validation accuracy are as follows:
![screenshot from 2019-02-19 09-37-30](https://user-images.githubusercontent.com/32355443/52992407-edd56700-3429-11e9-91a3-97c35cb9df4e.png)
But the validation accuracy reported in the paper is 70.57%. There is a big gap. Did I miss something that is important to the final performance of PointCNN?
